### DISCUSSION TITLE
RE4 DLSS Upscale + Reshade

### DISCUSSION THREAD

**Discussion by snoopi85** [praydog/REFramework#914] [General Questions]
Hi, is it possible to use the DLSS Mod Upscale option in RE4 remake and also use Reshade?

When ever I tick the 'Upscale' option in the Reframework Menu, it disables Reshade effects, but automatically enables if I untick 'Upscale'.

Any help appreciated, thanks.

**Comment by ghost** [comment]
Rename dxgi.dll to d3d12.dll.

**Reply by snoopi85** [reply]
Thank you. Will try this when I get home.

**Comment by snoopi85** [comment]
OK got it working. You are a legend sir, thank you so much!
